### PR TITLE
[KAS-5]: Make Login Screen

### PR DESCRIPTION
**Pull request by RiffatiKhan**
## Jira Story Link: [KAS-5](https://leezeit.atlassian.net/browse/KAS-5)
**Description** 
1) Login Signup screen has been designed
2) Design UI programmatically
3) Buttons are clickable but do not perform any action yet.

**How to test this**?
1) Run the and click Signup or Login Button
2) Print statement is added in order to test the buttons

### CODE REVIEW

File: LeezeIt/LeezeIt/Buttons.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+           super.init(coder: aDecoder)
+           setupButton()
+    }
+    
+    func setupButton() {
+        setTitleColor(.white, for: .normal)
+        backgroundColor            = .systemMint
```
Comment (Harrisali93) [inline LeezeIt/LeezeIt/Buttons.swift]: any specific reason for spacing???

File: LeezeIt/LeezeIt/Buttons.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        layer.cornerRadius         = 10
+        titleLabel?.font = UIFont.boldSystemFont(ofSize: 24)
+
```
Comment (Harrisali93) [inline LeezeIt/LeezeIt/Buttons.swift:27]: remove extra space

File: LeezeIt/LeezeIt/ViewController.swift [hunk 3]
```diff
@@ -7,13 +7,131 @@
...
+    let buttonStackView:  UIStackView = {
+        let stackView = UIStackView()
+        stackView.distribution = .fillEqually
+        stackView.axis = .vertical
+        stackView.spacing = 30
+        return stackView
+    }()
+    
+    var logoImage: UIImageView = {
+        let lztImage =  UIImageView(image: .init(named: "L"))
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift]: please fix your naming convention, change the name of the constant as well as the name of the image

File: LeezeIt/LeezeIt/ViewController.swift [hunk 4]
```diff
@@ -7,13 +7,131 @@
...
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        view.addSubview(parentStackView)
+        parentStackViewConst()
+        
+    } 
+    
+
+
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift:49]: remove extra white space

File: LeezeIt/LeezeIt/ViewController.swift [hunk 5]
```diff
@@ -7,13 +7,131 @@
...
+    
+    @objc  func logIn(_ sender: UIButton) {
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift]: there are two spaces between @obj and func, fix it

File: LeezeIt/LeezeIt/ViewController.swift [hunk 6]
```diff
@@ -7,13 +7,131 @@
...
+    }
+    
+    @objc  func signUP(_ sender: UIButton) {
+        print("signUp Button Tapped")
+        
     }
+}
 
 
+extension ViewController {
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift:60]: don't write short forms, write Constraints instead of Const

**Review by haroonmaqsoodch — COMMENTED** [review]
Buttons don't show up on ipod 7th gen

File: LeezeIt/LeezeIt/Buttons.swift [hunk 7]
```diff
@@ -0,0 +1,30 @@
+//
+//  Buttons.swift
+//  LeezeIt
+//
+//  Created by Riffat Khan on 9/2/22.
+//
+
+import UIKit
+
+class SurveyButton: UIButton {
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/Buttons.swift]: Come up with a better class name

File: LeezeIt/LeezeIt/ViewController.swift [hunk 8]
```diff
@@ -7,13 +7,131 @@
...
+        }
+    }
 }
 
+
+
+
+
+
+
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift]: remove the extra space please

Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/ViewController.swift]: resolved

File: LeezeIt/LeezeIt/Buttons.swift (hunk 7 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/Buttons.swift]: resolved

File: LeezeIt/LeezeIt/ViewController.swift (hunk 6 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/ViewController.swift:60]: resolved

File: LeezeIt/LeezeIt/ViewController.swift (hunk 5 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/ViewController.swift]: resolved

File: LeezeIt/LeezeIt/ViewController.swift (hunk 4 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/ViewController.swift:49]: resolved

File: LeezeIt/LeezeIt/ViewController.swift (hunk 3 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/ViewController.swift]: resolved

File: LeezeIt/LeezeIt/Buttons.swift (hunk 2 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/Buttons.swift:27]: resolved

File: LeezeIt/LeezeIt/Buttons.swift (hunk 1 above)
Comment (RiffatiKhan) [reply LeezeIt/LeezeIt/Buttons.swift]: resolved

File: LeezeIt/LeezeIt/ViewController.swift (hunk 6 above)
Comment (haroonmaqsoodch) [reply LeezeIt/LeezeIt/ViewController.swift:60]: the image that you have in the assets, change the name of that "L" isn't a good name

**Comment by Harrisali93** [comment]
looks good to me

**Review by Harrisali93 — DISMISSED** [review]

File: LeezeIt/LeezeIt/ViewController.swift [hunk 9]
```diff
@@ -7,13 +7,116 @@
...
+        parentStackViewConstraint()
+    }
+    
+    func addButtonsToStackView() {
+        let numOfButtons = 2
+        
+        for i in 1...numOfButtons {
+            
+            let button = CreateButton()
+
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift:110]: remove spaces

File: LeezeIt/LeezeIt/ViewController.swift [hunk 10]
```diff
@@ -7,13 +7,116 @@
...
+        leezeitImageStackView.centerXAnchor.constraint(equalTo: parentStackView.centerXAnchor).isActive               = true
+        leezeitImageStackView.topAnchor.constraint(equalTo: parentStackView.bottomAnchor, constant: -10).isActive     = true
+        leezeitImageStackView.addArrangedSubview(leezeitImage)
+        
+        leezeitImage.translatesAutoresizingMaskIntoConstraints                                               = false
+        leezeitImage.heightAnchor.constraint(equalTo: parentStackView.widthAnchor, multiplier: 0.4).isActive = true
+    }
+    
+    func buttonStackViewConstraint() {
+        buttonStackView.translatesAutoresizingMaskIntoConstraints                                                          = false
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/ViewController.swift:84]: remove extra space in all the methods

**Review by haroonmaqsoodch — APPROVED** [review]

**Review by Harrisali93 — APPROVED** [review]

**Pull request merged**